### PR TITLE
Accessibility card

### PR DESCRIPTION
**Pull request by mhewson**

### CODE REVIEW

File: docs/components/card/README.md [hunk 1]
```diff
@@ -120,10 +111,10 @@ CdrCards should always be used to link to other content, and the `cdr-card__link
 
 ## Accessibility
 
-To ensure that usage of this component complies with accessibility guidelines, do the following:
+Many WCAG requierments are contextual to their implementation. To ensure that usage of this component complies with accessibility guidelines you are responcible for the following:
```
Comment (cowills) [inline docs/components/card/README.md:114]: "responsible"

what happened to the article/section/div info? is that not applicable?

Comment (mhewson) [reply docs/components/card/README.md:114]: the component does not support it yet - so I backlogged it :P -man my spelling game is taking a nosedive

Comment (cowills) [reply docs/components/card/README.md]: what do you mean by "support it"? wouldn't the consumer just wrap the card content in that tag?

Comment (mhewson) [reply docs/components/card/README.md]: well the card as it is auto uses the article tag - I think we will want to update this to something like text where the tag defaults to article but can be set to section or div

File: docs/components/card/README.md [hunk 2]
```diff
@@ -120,10 +119,13 @@ CdrCards should always be used to link to other content, and the `cdr-card__link
 
 ## Accessibility
 
-To ensure that usage of this component complies with accessibility guidelines, do the following:
+Many WCAG requierments are contextual to their implementation. To ensure that usage of this component complies with accessibility guidelines you are responsible for the following:
 
-- TODO: something
-- TODO: something else
+- Use an `<article>` tag if the content includes a heading AND the content would be as useful on another web page
```
Comment (cowills) [inline docs/components/card/README.md:124]: i think we should clarify here that we mean that tag prop, so that people don't wrap the card content

**Pull request merged**